src/test/java/parser/ParserTest.java

```diff
@@ -40,6 +41,11 @@ public void getUserCommand_deleteManga_parsedCorrectly() {
         parseInputAssertCommandType("delete -a \"test\" -m \"test\"", DeleteMangaCommand.class);
     }
 
+    @Test
+    public void getUserCommand_addAuthor_parsedCorrectly() {
+        parseInputAssertCommandType("add -a \"Kubo Tite\"", AddAuthorCommand.class);
+    }
+
```

Comment:
nit: A suggestion would be to adhere to the style of command. For instance, in getUserCommand_deleteManga_parsedCorrectly(), the term "test" is used, rather than actual names or titles.